lib/imba/dom/manager.js

```diff
@@ -16,7 +16,7 @@ Imba.TagManagerClass.prototype.mounted = function (){
 };
 
 Imba.TagManagerClass.prototype.insert = function (node,parent){
-	this._inserts++;
+	if (node && node.querySelector && (node.mount || node.querySelector(".__mount"))) { this._tryMounting = true };
```

Comment:
Essentially calling querySelector whenever a node is inserted somewhere in a tag tree in Imba will introduce a _hefty_ performance penalty, so we need to find a different (more optimized) approach. I will look at the test you added and try to think about some more performant ways of dealing with this :)

Reply:
Yes, that is what I thought. But I couldn't think of any other way, so for now I will continue with this implementation until something better is found.